### PR TITLE
Updating and simplifying nest integration documentation structure

### PR DESCRIPTION
**Pull request by allenporter**
## Proposed change

Updating nest integration instructions.  The primary goals are:
- Reduce the overall amount of words describing the state of support of various devices.
- Give an overview of the supported devices in a table, since the number of devices and various capabilities has grown significantly in the last year. In particular, its hard for users to understand various states of streams, events supported, and which can actually capture media in response to events. This mirrors an existing google documentation table, but explains device/API capabilities in terms of home assistant features.
- Clarify differences between RTSP and WebRTC stream support
- Also call out Nest Hub Max known limitations since this is a common community forum issue. (I'm in contact with the nest API tech lead)
- Re-order Climate and Camera so that the camera related features flow together better (Camera, then Device Triggers, then Media Source)
- Other whitespace fixes

Follow up documentation improvements planned:
[ ] Restructure / move troubleshooting lower in the page e.g. collapsing logging
[ ] Move event automation example to use new media source API
[ ] Example for capturing an mp4 clip

## Type of change

- [X] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/nest.markdown [hunk 1]
```diff
@@ -23,40 +23,21 @@ ha_platforms:
...
-- Thermostat Devices
-  - Every thermostat is exposed as a `climate` entity
-  - A Temperature `sensor` entity. Note: Additional Nest Temperature Sensors are not supported by the SDM API.
-  - A Humidity `sensor` entity.
-  - Example devices: All Google Nest Thermostat models
-- Display, Camera, and Doorbell Devices
-  - The camera live stream is available as a `camera` entity
-  - Device Triggers for use in automations such as Person detected, Motion detected and Doorbell pressed
-  - Example devices: All wired & battery Google Nest Cam models, wired & battery Nest Doorbells, and Google Nest Hub Max.
+Cameras and Doorbells use [Automation and Device Triggers](#automation-and-device-triggers) for events and a [Media Source](#media-source) for capturing media iamges on supported devices. Other device types like Smoke and CO Alarms or Security systems are not currently supported by the SDM API.
```
Comment (frenck) [inline source/_integrations/nest.markdown]:
```suggestion
Cameras and Doorbells use [Automation and Device Triggers](#automation-and-device-triggers) for events and a [Media Source](#media-source) for capturing media images on supported devices. Other device types like Smoke and CO Alarms or Security systems are not currently supported by the SDM API.
```

File: source/_integrations/nest.markdown [hunk 2]
```diff
@@ -23,40 +23,21 @@ ha_platforms:
...
 
 You are in control of the information and capabilities exposed to Home Assistant. You can authorize a single device, multiple devices, or different levels of functionality such as motion events, live streams, for any particular device. The integration is flexible enough to adapt based on what you allow.
 
-Others devices like Smoke and CO Alarms or Security systems are not currently
-supported by the SDM API.
-
-The full detailed instructions for account setup are available in the [Device Access Registration](https://developers.google.com/nest/device-access/registration) Quick Start Guide. The instructions below are included to make this complex setup process a bit easier to follow.
+<div class='note'>
+The Nest Smart Device Management (SDM) API <b>requires a US$5 fee</b>.
+</div>
```
Comment (frenck) [inline source/_integrations/nest.markdown]:
```suggestion
<div class='note'>

The Nest Smart Device Management (SDM) API **requires a US$5 fee**.

</div>
```

File: source/_integrations/nest.markdown [hunk 3]
```diff
@@ -362,28 +333,35 @@ All Google Nest Thermostat models have traits exposed from the SDM API. The init
...
 
-The Nest integration makes [device triggers](/docs/automation/trigger/#device-triggers) available to enable automation
-in Home Assistant. You should review the [Automating Home Assistant](/getting-started/automation/) getting started guide on automations or the [Automation](/docs/automation/) documentation for full details.
+## Camera
 
-![Screenshot Device Triggers](/images/integrations/nest/device_triggers.png)
+Home Assistant supports all SDM API features. However, every Camera or Doorbell device has a different set of built-in capabilities. A Camera device has one of the following live stream types:
+
+  * **RTSP**: These devices have an HLS stream served by the Home Assistant Core. These cameras support server side `camera` services like stream recording, or image preview. See [Low Latency HLS](/integrations/stream#ll-hls) as a great option to enable to reduce stream latency.
+  * **WebRTC**: These devices support direct browser to camera communication, and a super low latency stream. Live image previews and the `camera` services like stream recording are *not supported*.
```
Comment (frenck) [inline source/_integrations/nest.markdown]:
```suggestion
- **RTSP**: These devices have an HLS stream served by the Home Assistant Core. These cameras support server-side `camera` services like stream recording or image preview. See [Low Latency HLS](/integrations/stream#ll-hls) as a great option to enable to reduce stream latency.
- **WebRTC**: These devices support direct browser to camera communication and a super low latency stream. Live image previews and the `camera` services like stream recording are *not supported*.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by allenporter** [comment]
Thank you for the fast review. I looked again locally and the commits look good, thanks.

**Review by frenck — APPROVED** [review]
Thanks, @allenporter 👍

**Pull request merged**